tests/components/plugwise/test_init.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for the Plugwise Climate integration."""
+
+import asyncio
+
+from Plugwise_Smile.Smile import Smile
+
+from tests.components.plugwise.common import async_init_integration
+
+
+async def test_smile_unauthorized(hass, mock_smile_unauth):
+    """Test failing unauthorization by Smile."""
+    assert not await async_init_integration(hass, mock_smile_unauth)
+
+
+async def test_smile_error(hass, mock_smile_error):
+    """Test server error handling by Smile."""
+    assert not await async_init_integration(hass, mock_smile_error)
```

Comment:
We want to also assert the config entry state here. It should be `ENTRY_STATE_SETUP_RETRY`.

Reply:
Modified accordingly with below comment